src/components/EntityModal/index.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import Modal from '@material-ui/core/Modal';
+import ArronaxIcon from '../ArronaxIcon';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const ModalWrapper = styled(Modal)``;
+
+const ScrollContainer = styled.div`
+  width: 100%;
+  height: 100%;
+  overflow-y: auto;
+  padding: 77px 0;
+`;
+
+const ModalContainer = styled.div`
+  background-color: #ffffff;
+  outline: none;
+  position: relative;
+  padding: 27px 30px 30px 30px;
+  margin: 0 auto;
+  width: 798px;
+  min-height: 100%;
+`;
+
+const ListContainer = styled.div`
+  width: 100%;
+`;
+
+const CloseIcon = styled(ArronaxIcon)`
```

Comment:
add a "Close" button at the bottom in the same style as "Reset" on the filter view